CustomDynamicFont/Sources/Presentation/Modules/iOS10/IOS10ViewController.swift

```diff
@@ -43,7 +43,15 @@ extension IOS10ViewController {
     }
 
     private func bindViewModel() {
+        viewModel.input.setOS(version: .ios10)
         viewModel.output.title.drive(rx.title)
             .disposed(by: disposeBag)
+        viewModel.output.overrideFontSize
+            .asObservable()
+            .withUnretained(self)
+            .subscribe { owner, value in
+                owner.fontSize = value
+            }
```

Comment:
Optional: we make this `.subscribe { owner, value in owner.fontSize = value }` one line as it doesn't exceed the line length limit 🙏